tests/sanity/test.wake.template

```diff
@@ -0,0 +1,16 @@
+
+def sanityScalaModule =
+  def scalaVersion = chisel3ScalaModule.getScalaModuleScalaVersion
+  makeScalaModule "sanity" here scalaVersion
+  | setScalaModuleDeps (chisel3ScalaModule, Nil)
+  | setScalaModuleScalacOptions ("-Xsource:2.11", Nil)
+
+def test Unit =
```

Comment:
Could we write a single sentence comment describing the test intent? Something simple like "Checks that a Verilog file can be compiled" would be fine. I just want to guard against something that I find often in codebases where it's not exactly obvious what was being tested.

Reply:
Yeah that makes sense, will add a comment and rebase